README.md

```diff
@@ -1,6 +1,6 @@
 # fml (Fast MARC Library)
 
-fml is a Go library for parsing MARC 21 formatted data.
+fml is a Go library for parsing MARC 21 formatted data. The library interface should still be considered unstable and may change in backwards incompatible ways.
```

Comment:
Hold on their buddy, I'm not sure I'd have allowed this in our project if I knew about this ;)

Reply:
The first change will be to remove the ability to parse MARC.